gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/exclusion/functions/LeftWhenRightIsNull.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright © 2014 - 2017 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.exclusion.functions;
+
+import org.apache.flink.api.common.functions.FlatJoinFunction;
+import org.apache.flink.util.Collector;
+
+/**
+ * Returns the left side if the right side is null.
+ *
+ * @param <E> an object type
+ */
+public class LeftWhenRightIsNull<E> implements FlatJoinFunction<E, E, E> {
```

Comment:
this is a convenience function and should be moved to `org.gradoop.flink.model.impl.functions.utils`